web/react/components/login.jsx

```diff
@@ -31,11 +32,11 @@ var FindTeamDomain = React.createClass({
         state.server_error = "";
         this.setState(state);
 
-        client.findTeamByDomain(domain,
+        client.findTeamByURLId(urlId,
             function(data) {
                 console.log(data);
                 if (data) {
-                    window.location.href = window.location.protocol + "//" + domain + "." + utils.getDomainWithOutSub();
+					window.location.href = req.getResponseHeader(Constants.TEAM_URL_HEADER) + '/channels/town-square';
```

Comment:
See comment above. But not having it in the store allows someone to access the site from different domains. Such as IP address or domain.